tests/testthat/test-dummy.R

```diff
@@ -151,6 +151,137 @@ test_that("create all dummy variables", {
   )
 })
 
+test_that("make sure contrasts argument work", {
```

Comment:
Can you try an example with `hardhat::contr_one_hot()` or some other external contrast function?

Reply:
done in https://github.com/tidymodels/recipes/pull/1484/commits/f49dac1229d4c982339e542211beefca179b1851